app/src/main/java/com/framgia/soundclound/screen/favourite/FavoriteTrackAdapter.java

```diff
@@ -0,0 +1,95 @@
+package com.framgia.soundclound.screen.favourite;
+
+import android.databinding.DataBindingUtil;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
```

Comment:
ERROR: Unused import: android.view.View. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports